### PR TITLE
RDoc-1529/1545 opening a session

### PR DESCRIPTION
**Pull request by morhilai**

### CODE REVIEW

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown [hunk 1]
```diff
@@ -1,83 +1,65 @@
...
 
-To open a synchronous session, use the `OpenSession` method from `DocumentStore`. If you prefer working in an asynchronous manner, use `OpenAsyncSession`.
+* Always open a new session object in a `using` statement so that the resources allocated to it can be released after you're done with it. Alternatively, you can call `.Dispose()` on it. 
+(The session implements [IDisposable](https://docs.microsoft.com/en-us/dotnet/api/system.idisposable?view=netframework-4.7.2))  
 
-## Syntax
+* There are two kinds of session: **sync** and **async**. Both can be opened from the Document Store, with `.OpenSession()` and `.OpenAsyncSession()` respectively.  
 
-There are three overloads of `OpenSession / OpenAsyncSession` methods
+* `OpenSession` and `OpenAsyncSession` each have three overloads that create sessions with different **session options**.
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: Replace overview bullets with:
=>

* A session object is obtained from the {Document Store](..)

* A Session can operate Synchronously or Asynchronously
  *  `OpenSession()` - Open a session for a **Synchrounous** mode of operation
  *  `OpenAsyncSession()` - Open a session for **Asychrounous** mode of operation

* Various sessions options can be configured using the `SessionsOptions` object

* Be sure to wrap the Session variable with a 'using' statement to ensure proper disposal

* If no database is specified then the default database (stored in the Document Store) is assumed

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown [hunk 2]
```diff
@@ -1,83 +1,65 @@
...
+
+* In this page:  
+  * [Syntax](../../client-api/session/opening-a-session#syntax)  
+  * [SessionOptions](../../client-api/session/opening-a-session#sessionoptions)  
+{NOTE/}
+
+---
+{PANEL:Syntax}
+
+There are three overloads each of `OpenSession()`/`OpenAsyncSession()`:
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: `OpenSession()` / `OpenAsyncSession()` have three overloads:

File: Documentation/4.1/Samples/csharp/Raven.Documentation.Samples/ClientApi/Session/OpeningSession.cs [hunk 3]
```diff
@@ -27,54 +27,74 @@ private interface IFoo2
         private interface IFoo
         {
             #region open_session_1
-            // Open session for a 'default' database configured in 'DocumentStore'
+            // First overload
```
Comment (Danielle9897) [inline Documentation/4.1/Samples/csharp/Raven.Documentation.Samples/ClientApi/Session/OpeningSession.cs]: Remove above comment

File: Documentation/4.1/Samples/csharp/Raven.Documentation.Samples/ClientApi/Session/OpeningSession.cs [hunk 4]
```diff
@@ -27,54 +27,74 @@ private interface IFoo2
         private interface IFoo
         {
             #region open_session_1
-            // Open session for a 'default' database configured in 'DocumentStore'
+            // First overload
+            // Open session for the default database configured in `DocumentStore.Database`
```
Comment (Danielle9897) [inline Documentation/4.1/Samples/csharp/Raven.Documentation.Samples/ClientApi/Session/OpeningSession.cs]: => Open a session ....

File: Documentation/4.1/Samples/csharp/Raven.Documentation.Samples/ClientApi/Session/OpeningSession.cs [hunk 5]
```diff
@@ -27,54 +27,74 @@ private interface IFoo2
         private interface IFoo
         {
             #region open_session_1
-            // Open session for a 'default' database configured in 'DocumentStore'
+            // First overload
+            // Open session for the default database configured in `DocumentStore.Database`
             IDocumentSession OpenSession();
 
+            // Second overload
```
Comment (Danielle9897) [inline Documentation/4.1/Samples/csharp/Raven.Documentation.Samples/ClientApi/Session/OpeningSession.cs]: Remove above comment

File: Documentation/4.1/Samples/csharp/Raven.Documentation.Samples/ClientApi/Session/OpeningSession.cs [hunk 6]
```diff
@@ -27,54 +27,74 @@ private interface IFoo2
         private interface IFoo
         {
             #region open_session_1
-            // Open session for a 'default' database configured in 'DocumentStore'
+            // First overload
+            // Open session for the default database configured in `DocumentStore.Database`
             IDocumentSession OpenSession();
 
+            // Second overload
             // Open session for a specified database
```
Comment (Danielle9897) [inline Documentation/4.1/Samples/csharp/Raven.Documentation.Samples/ClientApi/Session/OpeningSession.cs]: => Open a session for the specified database

File: Documentation/4.1/Samples/csharp/Raven.Documentation.Samples/ClientApi/Session/OpeningSession.cs [hunk 7]
```diff
@@ -27,54 +27,74 @@ private interface IFoo2
...
-            // Open session for a 'default' database configured in 'DocumentStore'
+            // First overload
+            // Open session for the default database configured in `DocumentStore.Database`
             IDocumentSession OpenSession();
 
+            // Second overload
             // Open session for a specified database
             IDocumentSession OpenSession(string database);
 
+            // Third overload
```
Comment (Danielle9897) [inline Documentation/4.1/Samples/csharp/Raven.Documentation.Samples/ClientApi/Session/OpeningSession.cs]: Remove above comment

File: Documentation/4.1/Samples/csharp/Raven.Documentation.Samples/ClientApi/Session/OpeningSession.cs [hunk 8]
```diff
@@ -27,54 +27,74 @@ private interface IFoo2
...
+            // Open session and pass it a preconfigured SessionOptions object
```
Comment (Danielle9897) [inline Documentation/4.1/Samples/csharp/Raven.Documentation.Samples/ClientApi/Session/OpeningSession.cs]: => Open a session ...

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown [hunk 9]
```diff
@@ -1,83 +1,65 @@
...
+<br/>
 
-The first method is an equivalent of doing
+The first two overloads simply call the third overload and pass it a new `SessionOptions` object.
 
 {CODE-TABS}
 {CODE-TAB:csharp:Sync open_session_2@ClientApi\Session\OpeningSession.cs /}
 {CODE-TAB:csharp:Async open_session_2_1@ClientApi\Session\OpeningSession.cs /}
 {CODE-TABS/}
+{PANEL/}
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: Remove all above part (lines 28-33) - redundant

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown [hunk 10]
```diff
@@ -1,83 +1,65 @@
...
 
-The second method is an equivalent of doing
-
-{CODE-TABS}
-{CODE-TAB:csharp:Sync open_session_3@ClientApi\Session\OpeningSession.cs /}
-{CODE-TAB:csharp:Async open_session_3_1@ClientApi\Session\OpeningSession.cs /}
-{CODE-TABS/}
-
-| Parameters | | |
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: Add this table back.
Provide titles for all columns (i.e. Paramerer | Type | Description)

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown [hunk 11]
```diff
@@ -1,83 +1,65 @@
...
-| ------------- | ------------- | ----- |
-| **options** | `OpenSessionOptions` | Options **containing** information such as **name of database** and **RequestExecutor**. |
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: 1. The type: 
    `OpenSessionOptions` => `SessionOptions`
2.  The description: 
     => An object with sessions configuraton options. See details [below](..)

Comment (Danielle9897) [reply Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: 3. Add an entry in the table with:

| database | string | The session will operate on this database, overriding the default database set in the document store.

4. place it above the 'options' entry...

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown [hunk 12]
```diff
@@ -1,83 +1,65 @@
...
-
-| Return Value | |
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: Add this table back.
Provide titles for all columns (i.e. Return Value | Description)

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown [hunk 13]
```diff
@@ -1,83 +1,65 @@
...
-{CODE-TABS/}
-
-## Example II - Disabling Entities Tracking
+{PANEL:SessionOptions}
 
-{CODE-TABS}
-{CODE-TAB:csharp:Sync open_session_tracking_1@ClientApi\Session\OpeningSession.cs /}
-{CODE-TAB:csharp:Async open_session_tracking_2@ClientApi\Session\OpeningSession.cs /}
-{CODE-TABS/}
+The `SessionOptions` object is a property of `session` which contains various options to configure the session's behavior:
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: => 
The `SessionOptions` object contains various options to configure the session's behavior.

(It is Not a property of the Session....)

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown [hunk 14]
```diff
@@ -1,83 +1,65 @@
...
-{CODE-TABS}
-{CODE-TAB:csharp:Sync open_session_tracking_1@ClientApi\Session\OpeningSession.cs /}
-{CODE-TAB:csharp:Async open_session_tracking_2@ClientApi\Session\OpeningSession.cs /}
-{CODE-TABS/}
+The `SessionOptions` object is a property of `session` which contains various options to configure the session's behavior:  
 
-## Remarks
+| Option | Type | Description | Default Value |
+| --- | --- | --- | --- |
+| **Database** | string | The name of the database the session should operate on | `null` - the session operates on the [default database](../../client-api/setting-up-default-database) |
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: The name of .... => The session will operate on this database,  overriding the Default Database.

Comment (Danielle9897) [reply Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: default database => Default Database (in the Default Value column)

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown [hunk 15]
```diff
@@ -1,83 +1,65 @@
...
+| **NoTracking** | bool | Whether the session should keep track of changes to `stored` or `loaded` entities. [How to Disable Tracking](../../client-api/session/configuration/how-to-disable-tracking) | `false` |
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: => Whether the session should track changes made to all entities that it has either loaded, stored, or queried on.
      `<br>`. See [Disable Tracking Example].

NOTE the usage of `<br>` within the text !!
This causes the line to actually break nicely within the table text.

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown [hunk 16]
```diff
@@ -1,83 +1,65 @@
...
+| **NoCaching** | bool | Whether the session should cache responses. [How to Disable Caching](../../client-api/session/configuration/how-to-disable-caching) | `false` |
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: 1. => Whether the session should cache the server responses.
2. Use the  `<br>` to break text within

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown [hunk 17]
```diff
@@ -1,83 +1,65 @@
...
+| **RequestExecutor** | `RequestExecutor` | _(Advanced)_ The request executor the session should use | `null` - the default request executor is used |
+| **TransactionMode** | `TransactionMode` | The two transaction modes are `SingleNode` and `ClusterWide`. Learn more about [Cluster-Wide Transactions](../../server/clustering/cluster-transactions) | `SingleNode` |
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: The two.... 
=> 
Specify the session's transaction mode (`SingleNode` /  `ClusterWide`).
 `<br>` Learn more in [Cluster-Wide Transactions](...)

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown [hunk 18]
```diff
@@ -1,83 +1,65 @@
...
 
-## Syntax
+* There are two kinds of session: **sync** and **async**. Both can be opened from the Document Store, with `.OpenSession()` and `.OpenAsyncSession()` respectively.  
 
-There are three overloads of `OpenSession / OpenAsyncSession` methods
+* `OpenSession` and `OpenAsyncSession` each have three overloads that create sessions with different **session options**.
+
+* In this page:  
+  * [Syntax](../../client-api/session/opening-a-session#syntax)  
+  * [SessionOptions](../../client-api/session/opening-a-session#sessionoptions)
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: Add another panel:

* Example

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown [hunk 19]
```diff
@@ -1,83 +1,65 @@
...
+| **Database** | string | The name of the database the session should operate on | `null` - the session operates on the [default database](../../client-api/setting-up-default-database) |
+| **NoTracking** | bool | Whether the session should keep track of changes to `stored` or `loaded` entities. [How to Disable Tracking](../../client-api/session/configuration/how-to-disable-tracking) | `false` |
+| **NoCaching** | bool | Whether the session should cache responses. [How to Disable Caching](../../client-api/session/configuration/how-to-disable-caching) | `false` |
+| **RequestExecutor** | `RequestExecutor` | _(Advanced)_ The request executor the session should use | `null` - the default request executor is used |
+| **TransactionMode** | `TransactionMode` | The two transaction modes are `SingleNode` and `ClusterWide`. Learn more about [Cluster-Wide Transactions](../../server/clustering/cluster-transactions) | `SingleNode` |
 
-{DANGER:Important}
-**Always remember to release session allocated resources after usage by invoking the `Dispose` method or wrapping the session object in the `using` statement.**
-{DANGER/}
+{PANEL/}
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: Add a panel with a short example (sync & async), i.e.:

```
using (var store = new DocumentStore())
{    
    // Open a session in synchronous operation mode for cluster-wide transactions

    SessionOptions options = new SessionOptions  {
          Database = "your_database_name",       
          TransactionMode = TransactionMode.ClusterWide
    };

    using (IDocumentSession session =  store.OpenSession(options))
    {
       //   Run your business logic:
       //   
       //   Store documents
       //   Load and Modify documents
       //   Query indexes & collections 
       //   Delete documents
       //   .... etc.
   
       session.SaveChanges();
    }
}

```

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown (hunk 1 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: Done

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown (hunk 2 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: Done

File: Documentation/4.1/Samples/csharp/Raven.Documentation.Samples/ClientApi/Session/OpeningSession.cs (hunk 3 above)
Comment (morhilai) [reply Documentation/4.1/Samples/csharp/Raven.Documentation.Samples/ClientApi/Session/OpeningSession.cs]: Done

File: Documentation/4.1/Samples/csharp/Raven.Documentation.Samples/ClientApi/Session/OpeningSession.cs (hunk 4 above)
Comment (morhilai) [reply Documentation/4.1/Samples/csharp/Raven.Documentation.Samples/ClientApi/Session/OpeningSession.cs]: Done

File: Documentation/4.1/Samples/csharp/Raven.Documentation.Samples/ClientApi/Session/OpeningSession.cs (hunk 5 above)
Comment (morhilai) [reply Documentation/4.1/Samples/csharp/Raven.Documentation.Samples/ClientApi/Session/OpeningSession.cs]: Done

File: Documentation/4.1/Samples/csharp/Raven.Documentation.Samples/ClientApi/Session/OpeningSession.cs (hunk 6 above)
Comment (morhilai) [reply Documentation/4.1/Samples/csharp/Raven.Documentation.Samples/ClientApi/Session/OpeningSession.cs]: Done

File: Documentation/4.1/Samples/csharp/Raven.Documentation.Samples/ClientApi/Session/OpeningSession.cs (hunk 7 above)
Comment (morhilai) [reply Documentation/4.1/Samples/csharp/Raven.Documentation.Samples/ClientApi/Session/OpeningSession.cs]: Done

File: Documentation/4.1/Samples/csharp/Raven.Documentation.Samples/ClientApi/Session/OpeningSession.cs (hunk 8 above)
Comment (morhilai) [reply Documentation/4.1/Samples/csharp/Raven.Documentation.Samples/ClientApi/Session/OpeningSession.cs]: Done

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown (hunk 9 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: Done

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown (hunk 10 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: Done

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown (hunk 11 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: Done

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown (hunk 12 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: Done

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown (hunk 13 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: Done

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown (hunk 14 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: Done

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown (hunk 15 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: Done

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown (hunk 16 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: Done

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown (hunk 17 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: Done

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown (hunk 18 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: Done

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown (hunk 19 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: Done

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown (hunk 1 above)
Comment (Danielle9897) [reply Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: Various sessions options => Various session options
`SessionsOptions` => 'SessionOptions'

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown [hunk 20]
```diff
@@ -1,83 +1,81 @@
...
+
+| Option | Type | Description | Default Value |
+| --- | --- | --- | --- |
+| **Database** | string | The Session will operate on this database, overriding the Default Database. | `null` - the Session operates on the [Default Database](../../client-api/setting-up-default-database) |  
+| **NoTracking** | bool | Whether the Session should track changes made to all entities that it has either loaded, stored, or queried on. <br>See [Disable Tracking Example](../../client-api/session/configuration/how-to-disable-tracking) | `false` |  
+| **NoCaching** | bool | Whether the Session should cache the server responses. <br>[How to Disable Caching](../../client-api/session/configuration/how-to-disable-caching) | `false` |  
+| **RequestExecutor** | `RequestExecutor` | _(Advanced)_ The request executor the Session should use | `null` - the default request executor is used |  
+| **TransactionMode** | `TransactionMode` | Specify the Session's transaction mode (`SingleNode` / `ClusterWide`). <br>Learn more in [Cluster-Wide Transactions](../../server/clustering/cluster-transactions) | `SingleNode` |  
+
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: image

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown [hunk 21]
```diff
@@ -1,83 +1,81 @@
...
+{NOTE: }
+
+* A Session object is obtained from the [Document Store](../../client-api/what-is-a-document-store).  
+
+* A Session can operate Synchronously or Asynchronously.  
+  * `OpenSession()` - Open a Session for a **Synchrounous** mode of operation.  
+  * `OpenAsyncSession()` - Open a Session for **Asychrounous** mode of operation.  
+
+* Various Sessions options can be configured using the `SessionsOptions` object.
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: Various sessions options => Various session options
SessionsOptions => SessionOptions

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown (hunk 18 above)
Comment (Danielle9897) [reply Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: 'Example' is still missing

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown [hunk 22]
```diff
@@ -1,83 +1,81 @@
...
+
+* Various Sessions options can be configured using the `SessionsOptions` object.  
+
+* Be sure to wrap the Session variable with a 'using' statement to ensure proper disposal.  
+
+* If no database is specified then the Default Database (stored in the Document Store) is assumed.  
+
+* In this page:  
+  * [Syntax](../../client-api/session/opening-a-session#syntax)  
+  * [Session Options](../../client-api/session/opening-a-session#session-options)
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: 'Example' is still missing - see original comment

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown [hunk 23]
```diff
@@ -1,83 +1,81 @@
...
+
+| Option | Type | Description | Default Value |
+| --- | --- | --- | --- |
+| **Database** | string | The Session will operate on this database, overriding the Default Database. | `null` - the Session operates on the [Default Database](../../client-api/setting-up-default-database) |  
+| **NoTracking** | bool | Whether the Session should track changes made to all entities that it has either loaded, stored, or queried on. <br>See [Disable Tracking Example](../../client-api/session/configuration/how-to-disable-tracking) | `false` |  
+| **NoCaching** | bool | Whether the Session should cache the server responses. <br>[How to Disable Caching](../../client-api/session/configuration/how-to-disable-caching) | `false` |  
+| **RequestExecutor** | `RequestExecutor` | _(Advanced)_ The request executor the Session should use | `null` - the default request executor is used |  
+| **TransactionMode** | `TransactionMode` | Specify the Session's transaction mode (`SingleNode` / `ClusterWide`). <br>Learn more in [Cluster-Wide Transactions](../../server/clustering/cluster-transactions) | `SingleNode` |  
+
+Here is an example of using a `SessionOptions` object:
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: 1. To be placed in a separate panel (see 'in this page' comment)
2. => Here is an example of opening a Session using.....

Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: Done

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown (hunk 22 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: Done

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown (hunk 21 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: Done

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown (hunk 20 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: Done

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown [hunk 24]
```diff
@@ -1,83 +1,86 @@
...
+| **NoTracking** | boolean | Whether the Session should track changes made to all entities that it has either loaded, stored, or queried on. See [Disable Tracking Example](../../client-api/session/configuration/how-to-disable-tracking) | `false` |
+| **NoCaching** | boolean | Whether the Session should cache the server responses. See [Disable Caching Example](../../client-api/session/configuration/how-to-disable-caching) | `false` |
+| **RequestExecutor** | `RequestExecutor` | _(Advanced)_ The request executor the Session should use | `null` - the default request executor is used |
+| **TransactionMode** | `TransactionMode` | Specify the Session's transaction mode (`SingleNode` / `ClusterWide`). See [Cluster-Wide Transactions](../../server/clustering/cluster-transactions) | `SingleNode` |
+
+{PANEL/}
+
+{PANEL:Example}
+
+Here is an example of opening a Session using an `OpenSession`:
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: => Here is an example of opening a Session using a `SessionOptions` object
(not using an `OpenSession`)

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown [hunk 25]
```diff
@@ -1,83 +1,86 @@
...
+{PANEL/}
+
+{PANEL:Session Options}
+
+The `SessionOptions` object contains various options to configure the Session's behavior.
+
+| Option | Type | Description | Default Value |
+| --- | --- | --- | --- |
+| **Database** | string | The Session will operate on this database, overriding the Default Database. | `null` - the Session operates on the [Default Database](../../client-api/setting-up-default-database) |
+| **NoTracking** | boolean | Whether the Session should track changes made to all entities that it has either loaded, stored, or queried on. See [Disable Tracking Example](../../client-api/session/configuration/how-to-disable-tracking) | `false` |
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: Add `<br>`  before the See .....

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown [hunk 26]
```diff
@@ -1,83 +1,86 @@
...
+| **NoCaching** | boolean | Whether the Session should cache the server responses. See [Disable Caching Example](../../client-api/session/configuration/how-to-disable-caching) | `false` |
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: Add <br> before the See .....

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown [hunk 27]
```diff
@@ -1,83 +1,86 @@
...
+| **RequestExecutor** | `RequestExecutor` | _(Advanced)_ The request executor the Session should use | `null` - the default request executor is used |
+| **TransactionMode** | `TransactionMode` | Specify the Session's transaction mode (`SingleNode` / `ClusterWide`). See [Cluster-Wide Transactions](../../server/clustering/cluster-transactions) | `SingleNode` |
```
Comment (Danielle9897) [inline Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: Add <br> before the See .....

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown (hunk 25 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: Done

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown (hunk 26 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: Done

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown (hunk 27 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: Done

File: Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown (hunk 24 above)
Comment (morhilai) [reply Documentation/4.1/Raven.Documentation.Pages/client-api/session/opening-a-session.dotnet.markdown]: Done

**Pull request merged**